### ISSUE TITLE
Blockquotes Not Rendering

### ISSUE THREAD

**Issue by rafegoldberg** [erusev/parsedown#397]
Heyyo, loving Parsedown! That said, I've been unable to get v1.6 to correctly recognize and render blockquotes. Here's a snippet from my Markdown:

``` markdown
> A simple button object for use anywhere
```

Per the Markdown [syntax standard](https://daringfireball.net/projects/markdown/syntax#blockquote), one would expect this to render:

``` html
<blockquote>A simple button object for use anywhere</blockquote>
```

Instead I'm seeing the following: 

``` html
<p>> A simple button object for use anywhere</p>
```

I went through Parsedown's source code to see if I could fix it but it's a bit over my head. Please advise!

**Comment by erusev** [comment]
hm, can u reproduce this at http://parsedown.org/demo

**Comment by rafegoldberg** [comment]
hmmm, good question and strangely enough I can't! Seems to function exactly
as expected, which is great cause it means the issue is on my end
somewhere, but really that only confuses me even more!

On Sun, Mar 27, 2016 at 2:02 PM Emanuil Rusev <notifications@github.com>
wrote:

> hm, can u reproduce this at http://parsedown.org/demo
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/erusev/parsedown/issues/397#issuecomment-202144877>
>
-- 
--rg

**Comment by keyilan** [comment]
Make sure you're not doing something that's converting it to `&gt;`. This was my problem. All the `>` quotes had been converted to html entities in the database I was using.

**Comment by bartb** [comment]
I have the same problem... But only when I use a blockquote inside a div with markdown="1". Maybe that will help somebody.

**Comment by aidantwoods** [comment]
Since it didn't appear to be possible to reproduce this issue, I'm going to close this one – please reopen if you can reproduce this with the latest version though.

@bartb it may be the case that your issue related to markdown within HTML is fixed by the changes in #514, which will be included in the next release.
